<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>控制系统</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>source/iconn.ico</normaloff>source/iconn.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QPushButton" name="restartButton">
          <property name="text">
           <string>重置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="leftGroundButton">
          <property name="text">
           <string>左半球场</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="rightGroundButton">
          <property name="text">
           <string>右半球场</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Port</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="portLineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="serverButton">
            <property name="text">
             <string>Server打开</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="socketLineEditer"/>
        </item>
        <item>
         <widget class="Line" name="line_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>波特率</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="baudRateComboBox">
            <property name="currentText">
             <string>1200</string>
            </property>
            <item>
             <property name="text">
              <string>1200</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2400</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4800</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9600</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>19200</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>38400</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>57600</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>115200</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>数据位</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="dataBitsComboBox">
            <property name="currentText">
             <string>5</string>
            </property>
            <item>
             <property name="text">
              <string>5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>7</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>校验位</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="parityComboBox">
            <item>
             <property name="text">
              <string>No</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Even</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Odd</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Space</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Mark</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>停止位</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="stopBitsComboBox">
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1.5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QComboBox" name="comComboBox"/>
          </item>
          <item>
           <widget class="QPushButton" name="comRefreshButton">
            <property name="text">
             <string>刷新</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="comButton">
          <property name="text">
           <string>串口打开</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="comLineEdit"/>
        </item>
        <item>
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>X轴</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Y轴</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Theta角</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="xLineEdit">
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="yLineEdit">
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="thetaLineEdit">
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="handPositionButton">
          <property name="text">
           <string>手动设置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="shotButton">
          <property name="text">
           <string>弹射</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>733</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_3"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>重置</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
